admin/tests/integration/components/administration/organizations-import_test.js

```diff
@@ -0,0 +1,69 @@
+import { module, test } from 'qunit';
+import { render } from '@1024pix/ember-testing-library';
+import hbs from 'htmlbars-inline-precompile';
+import { triggerEvent } from '@ember/test-helpers';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import sinon from 'sinon';
+import Service from '@ember/service';
+import setupIntlRenderingTest from '../../../helpers/setup-intl-rendering';
+
+module('Integration | Component |  administration/organizations-import', function (hooks) {
+  setupIntlRenderingTest(hooks);
+  setupMirage(hooks);
+
+  test('it should be possible to create multiple organizations with csv import', async function (assert) {
```

Comment:
```suggestion
  test('it creates multiple organizations with csv import', async function (assert) {
```

Mais techniquement tout ce que le test teste c'est "it displays success notification if import succeeds".

Reply:
Oui